app/persistence/transaction/transaction.go

```diff
@@ -83,6 +83,7 @@ type Transaction struct {
 	EthTxStatus           string
 	EthHash               string
 	GasPriceGwei          string
+	Asset			      string
```

Comment:
Why is this showing it with such indentation

Reply:
This is the first time I am seeing it... The IDE displayed it as it should be.